config/application.rb

```diff
@@ -19,7 +20,7 @@ module DevelopmentApp
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.2
-    config.time_zone = "Europe/Paris"
+    config.time_zone = "Europe/Paris" unless Rails.env.test?
```

Comment:
```suggestion
    config.time_zone = "Europe/Paris"
```

time_zone should not be Europe/Paris ?